### DISCUSSION TITLE
Sunny Home Manager 2.0 auslesen

### DISCUSSION THREAD

**Discussion by tokotronic** [camueller/SmartApplianceEnabler#484] [Installation und Betrieb]
Hallo,

ich nutze zum Energiemanagement den Loxone Miniserver. Leider gibt es keine Verbindung zw. SHM und Loxone.
Kann ich SmartApplianceEnabler nutzen. um Sunny Home Manager per HTTP-Aufrug auszulesen? Z.B.
http://smartapplianceanabler:port/shm_aktuelle_leistung

VG
tokotronic

**Comment by camueller** [comment]
Nein
